src/main.cpp

```diff
@@ -260,6 +264,7 @@ void undoLastTurn() {
     turnList[turnCount-1] = -1;
     cellsState[cellIndex] = 0;
     turnCount--;
+    undoMusic();
```

Comment:
what if we call this `playUndoSound` to match the others?

Reply:
ok changed